### DISCUSSION TITLE
如何在程序运行时动态生成和映射实体类

### DISCUSSION THREAD

**Discussion by lostluna1** [dotnetcore/FreeSql#1898] [Q&A] [answered]
我不知道这个功能是否已经存在或者是我没找到, 我想在App运行时根据DB中的指定表生成实体类,然后在不重启App的情况下能够操作这些实体类

**Comment by 2881099** [comment, marked as answer]
https://freesql.net/guide/lowcode.html
都在这里

**Reply by lostluna1** [reply]
多谢大佬,回复太及时了
